source/includes/_products.md

```diff
@@ -271,7 +271,9 @@ curl https://api.zinc.io/v1/search?query=fish%20oil&page=1&retailer=amazon \
   -u <client_token>:
 ```
 
-Get search results from a retailer based on a query term. Results include product id, title, image url, number of reviews, star rating, and price.
+Get search results from a retailer based on a query term. Results include product id, title, image url, number of reviews, star rating, and price as well as how many results are shown of all available. 
+
+The api is optimized for fast responses so results are paged. That means that if there are lots of results returned by your query, the response will be paged. You can retrieve more results by executing the query again with an increased page number. For example, if querying the first page lists `showing.start` as 1 and `showing.end` as 48 with a total amount of results as `showing.of` = 30000, you can expect 625 pages in total (30000/48). However, keep in mind that query results are not being cached on our server, so executing a query again with another page number could return different results and will also be billed as each query causes new workload. That means your application should allow the showing object for changes with each query you execute.
```

Comment:
At the moment, it is unclear to the user how many items will return, because there is no way of setting the pagesize. Since the total amount of available items can vary between several executions, the user has to keep an eye on the values of the showing object. This point needs to be stressed more.